R/pkg/R/mllib.R

```diff
@@ -197,11 +197,10 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
-#'
 #' Makes predictions from a generalized linear model produced by glm() or spark.glm(),
 #' similarly to R's predict().
 #'
+#' @title predict
```

Comment:
Ok - then I'd say lets use the first line as the title convention for all our documentation.